bundles/model/org.openhab.model.script/src/org/openhab/model/script/internal/actions/TimerImpl.java

```diff
@@ -85,6 +102,26 @@ public boolean reschedule(AbstractInstant newTime) {
 		}
 	}
 	
+	// version 1.0.1 : allow to reschedule the cron timer
+	public boolean reschedule(String cronExpression) {
+		try {
+	        Trigger trigger = newTrigger()
+	        		.withSchedule( cronSchedule(cronExpression) )
+			        .build();
+    
+			scheduler.rescheduleJob(triggerKey, trigger);
+			this.triggerKey = trigger.getKey();
+			this.cancelled = false;
+			this.cronExpression = cronExpression;
+			this.terminated = false;
+			return true;
+		} catch (SchedulerException e) {
+			logger.warn("An error occured while rescheduling the job '{}': {}", new String[] { jobKey.toString(), e.getMessage() });
+			return false;
+		}	
```

Comment:
it looks like you've copied reschedule() and just replaced the Trigger creation. For the sake of Don't-Repeat-Yourself (DRY) could you please extract the Trigger-Creation from the methods and use just one resulting method which reschedules the Job properly